static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -36,12 +38,8 @@
     color: colour(neutral-2);
 }
 
-.crossword__clue--display-group-order {
-
-    padding-left: 0px;
-
-    //For grouped clues we display 11.12,23 accross instead of just a number
-    &:before {
-        position: initial;
-    }
+//For grouped clues we display 11.12,23 accross instead of just a number
+.crossword__clue--display-group-order:before {
+    padding-right: .875em; // 14px - Matches the width between double digit numbers and clue of non-grouped clues
```

Comment:
But if you need 14px exactly, don't worry about it :-) We also have a postprocessor that converts px to rem, so you're safe to use px!